app/com/linkedin/drelephant/spark/fetchers/SparkFetcher.scala

```diff
@@ -16,11 +16,9 @@
 
 package com.linkedin.drelephant.spark.fetchers
 
-import scala.async.Async
```

Comment:
Can you update the description of this PR with the motivation behind this change?